template_package/carma_package

```diff
@@ -50,6 +51,25 @@ if [[ ! -d ${package_parent_path} ]]; then
     exit 2
 fi
 
+plugin_type=$4
+
+if [ "$plugin_type" = "strategic" ] || [ "$plugin_type" = "tactical" ] || [ "$plugin_type" = "control" ]; then
```

Comment:
Huh, that's legal syntax? I had to look it up but I learned something today!

Reply:
Bash never ceases to confuse and confound 😄